### PR TITLE
New Crowdin updates

### PR DESCRIPTION
**Pull request by Picono435**

### CODE REVIEW

File: src/main/resources/langs/de-DE.yml [hunk 1]
```diff
@@ -15,45 +15,45 @@ ignore-action: "&cHey! Sei nicht faul und sprich mit dem NPC."
 choosed-job: "&aHerzlichen Glückwunsch! Du hast den Job %jobplayer_job%&a gewählt."
 accepted-work: "&aDu hast die Arbeit von %jobplayer_work% akzeptiert."
 no-salary: "&cDu hast nicht genug Gehalt, um es zu erhalten! Bitte gehe zur Arbeit ;("
-got-salary: "&aGute Arbeit! Du hast dein %jobplayer_salary% Gehalt erhalten!"
+got-salary: "Gute Arbeit! Du hast ein Gehalt von %jobplayer_salary% erhalten!"
```
Comment (NoJokeFNA) [inline src/main/resources/langs/de-DE.yml:18]:
```suggestion
got-salary: "&aGute Arbeit! Du hast ein Gehalt von %jobplayer_salary% erhalten!"
```

File: src/main/resources/langs/de-DE.yml [hunk 2]
```diff
@@ -15,45 +15,45 @@ ignore-action: "&cHey! Sei nicht faul und sprich mit dem NPC."
...
 finished-work: "&aHerzlichen Glückwunsch! Du hast deine Arbeit beendet. Benutze /jobs um mehr Informationen darüber zu erhalten."
 need-level: "&cHey! Du musst %a% tun, um diese Aktion auszuführen."
 no-args: "&cHey! Bitte verwende eine gültige Syntax."
-my-salary: "&aDu hast %jobplayer_salary%$ Gehalt."
+my-salary: "Du hast ein Gehalt von %jobplayer_salary%$."
```
Comment (NoJokeFNA) [inline src/main/resources/langs/de-DE.yml]:
```suggestion
my-salary: "&aDu hast ein Gehalt von %jobplayer_salary%$."
```

File: src/main/resources/langs/de-DE.yml [hunk 3]
```diff
@@ -15,45 +15,45 @@ ignore-action: "&cHey! Sei nicht faul und sprich mit dem NPC."
...
-  &e/jobs help &6- &eSehen Sie diese Nachrichten
-  &e/jobs wählen <job> &6- &eChooses your job
-  &e/jobs work &6- &eSiehe den Status/Akzeptieren Ihrer Arbeit
-  &e/jobs Gehalt &6- &ePrüfen Sie Ihr Gehalt
-  &e/jobs Abzug &6- &eRückzug Ihres Gehalts
-  &e/jobs leave &6- &eVerlässt den Job
+  &6Vollständige Liste der Befehle:
+  &e/jobs help &6- &eZeigt diese Meldung an
+  &e/jobs choose <job> &6- &eWählen eines Jobs
+  &e/jobs work &6- &eDein aktueller Status/Akzeptieren einer Arbeit
```
Comment (NoJokeFNA) [inline src/main/resources/langs/de-DE.yml:42]:
```suggestion
  &e/jobs work &6- &eDein aktueller Status-/ Deine aktuelle Akzeptanz einer Arbeit
```

**Review by NoJokeFNA — CHANGES_REQUESTED** [review]

**Comment by NoJokeFNA** [comment]
You forgot to push the two other suggestions

**Comment by Picono435** [comment]
Done :)

**Comment by NoJokeFNA** [comment]
:+1:

**Pull request merged**